### DISCUSSION TITLE
Does this de-obfuscate everything in the 'blob'

### DISCUSSION THREAD

**Discussion by danteali** [FuLoRi/Analyze-LastPassVaultGUI#13] [Q&A] [answered]
Excellent tool!

Not necessarily an issue but wondering if there is anything in the blob that is not de-obfuscated?
- I was expecting to see some output related to my 'Secure Notes' but only see the 'Saved Password' items.
- Or the notes field within the 'Saved Passwords'

I'm assuming that if I don't see it in the output then it's encrypted in the blob.

**Comment by FuLoRi** [comment, marked as answer]
Great questions!

This script **only** parses the contents of the "accounts" container. There are other things in the blob outside of that container that this script does not look at.

That said, _Secure Notes_ **are** stored in the "accounts" container. Look for entries where the URL is "http://sn" (ref: [this LastPass support article](https://support.lastpass.com/help/how-do-i-import-stored-data-into-lastpass-using-a-generic-csv-file)).

The _Notes_ in the site credential entries and the contents of _Secure Notes_ are both stored in the _Extra_ field.

**Reply by Petruchio96** [reply]
Another issue I'm worried about is file attachments on secure notes. According to ChatGPT(good source?) LastPass encrypts the file attachments as expected but the file names are not. So I'm assuming this is another piece of data that hackers could prioritize. Example, if they see a file in an encrypted vault titled "Backup Seed for Bitcoin wallet.txt" they would go after that vault.

**Comment by FuLoRi** [comment]
@danteali I'm going to close this issue.

I hope I answered your question with my previous comment.

Have a great day!
